parquet/src/test/scala/magnolify/parquet/test/ParquetTypeSuite.scala

```diff
@@ -80,6 +81,26 @@ class ParquetTypeSuite extends MagnolifySuite {
     test[UnsafeEnums]
   }
 
+//  test("ParquetDoc") {
+//    val pt = ensureSerializable(ParquetType[ParquetNestedDoc])
+//
+//    assert(pt .fieldAnnotations("pd") == "nested")
+//    assert(pt.fieldAnnotations("pd.i") == "integers")
+//    assert(pt.fieldAnnotations("pd.s") == "string")
+//    assert(pt.fieldAnnotations("i") == "integers")
+//    assert(pt.typeAnnotation.contains("Parquet with doc"))
+//  }
+
+//  test("ParquetDocWithNestedList") {
+//    val pt = ensureSerializable(ParquetType[ParquetNestedListDoc])
+//
+//    assert(pt.fieldAnnotations("pd") == "nested")
+//    assert(pt.fieldAnnotations("pd.i") == "integers")
+//    assert(pt.fieldAnnotations("pd.s") == "string")
+//    assert(pt.fieldAnnotations("i") == "integers")
+//    assert(pt.typeAnnotation.contains("Parquet with doc with nested list"))
+//  }
```

Comment:
This should be tested on the `FieldType`

Reply:
That is fixed in your commit